### DISCUSSION TITLE
Is requirements.txt correct with needed modules?

### DISCUSSION THREAD

**Discussion by DineshValor** [masterking32/MasterHamsterKombatBot#306] [Q&A]
```
requests[socks]
colorlog
Flask==3.0.3
gunicorn==23.0.0
```

Is any module missing or something wrong? Plz let me known! Need to host bot on server!

**Comment by tboy1337** [comment]
there is no requirements.txt with this project, the only modules that are needed currently are requests & colorlog eg```pip install requests colorlog```
